packages/shared/src/lib/reminderFields.ts

```diff
@@ -41,26 +65,29 @@ export const buildReminderFields = (today: Date = new Date()): ReminderFields =>
     };
 };
 
-const givingTuesdayCutOff = new Date('2023-11-27');
-
-const givingTuesdayIsActive = (date: Date): boolean => date <= givingTuesdayCutOff;
+const newYearsEve = new Date('2023-12-31');
+const givingTuesdayReminderStart = (date: Date): Date => {
+    return new Date(date.getFullYear(), 10, 1); // November 1st (provisional auto start date, to be confirmed)
```

Comment:
This year the US were keen to start showing this straight away, so "Remind me on Giving Tuesday" is already live on the site. Given that, I think it'd be worth either bringing this date forward a bit or waiting until tomorrow to merge, so that the correct messaging continues to be shown.